app/src/main/java/com/eveningoutpost/dexdrip/NavDrawerBuilder.java

```diff
@@ -16,13 +16,21 @@
 import com.eveningoutpost.dexdrip.Tables.CalibrationDataTable;
 import com.eveningoutpost.dexdrip.UtilityModels.CollectionServiceStarter;
 import com.eveningoutpost.dexdrip.UtilityModels.Experience;
+import com.eveningoutpost.dexdrip.UtilityModels.Pref;
 import com.eveningoutpost.dexdrip.stats.StatsActivity;
 import com.eveningoutpost.dexdrip.utils.DexCollectionType;
 import com.eveningoutpost.dexdrip.utils.Preferences;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import static com.eveningoutpost.dexdrip.utils.DexCollectionType.DexcomShare;
```

Comment:
Please remove these imports and use `DexCollectionType.hasSensorStartStopSupport` in line 114.